CMakeLists.txt

```diff
@@ -950,7 +950,9 @@ endif()
 
 foreach(ZLIB_INSTALL_LIBRARY ${ZLIB_INSTALL_LIBRARIES})
     target_include_directories(${ZLIB_INSTALL_LIBRARY} PUBLIC
-        ${CMAKE_CURRENT_BINARY_DIR} ${CMAKE_CURRENT_SOURCE_DIR})
+        "$<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR};${CMAKE_CURRENT_SOURCE_DIR}>"
+        "$<INSTALL_INTERFACE:${CMAKE_INSTALL_PREFIX}/include>"
```

Comment:
Using `${CMAKE_INSTALL_PREFIX}` is bad. See https://cmake.org/cmake/help/latest/manual/cmake-packages.7.html#creating-relocatable-packages

Reply:
Updated.